vars/imageMirror.txt

```diff
@@ -0,0 +1,69 @@
+# Image Mirror
+
+Image mirror allows you to move images from one image registry to another.
+
+Sample Usage:
+
+```
+stage ("Image Mirror") {
+  steps {
+    imageMirror(sourceSecret: "namespace-sourcesecret",
+                sourceRegistry: "https://source-registry.com",
+                sourceAPI: "source-api.com",
+                destinationSecret: "namespace-destinationsecret",
+                destinationRegistry: "https://destination-registry.com",
+                destinationAPI: "destination-api.com",
```

Comment:
I tested this and it works but would Registry and API addresses ever differ?
Maybe hide the different url format into the code?

Reply:
The only difference for me is `https://` but then we would be assuming the registry is https. I just wasnt sure that was a good approach. We could look for the protocol and not override it if it's there...

Reply:
Take a look at my changes... I think that may work well...